pkg/cmd/server/origin/master.go

```diff
@@ -517,6 +550,13 @@ func forbidden(reason string, w http.ResponseWriter, req *http.Request) {
 	fmt.Fprintf(w, "Forbidden: %q %s", req.RequestURI, reason)
 }
 
+// RunProjectAuthorizationCache starts the project authorization cache
+func (c *MasterConfig) RunProjectAuthorizationCache() {
+	// TODO: look at exposing a configuration option in future to control how often we run this loop
+	period, _ := time.ParseDuration("1s")
```

Comment:
ha, that must be a typo instead of 10.  how long do you want to wait?  it does nothing if the cache is populated but churn through N namespaces and N policies in a for loop making no requests.